scripts/setup-k8s-master.sh.in

```diff
@@ -34,25 +34,39 @@ test -n "{{Region}}"
 test -n "{{ClusterInfoBucket}}"
 test -n "{{ClusterDNSProvider}}"
 
+INSTANCE_ID=$(ec2metadata --instance-id)
+
+# Add this machine (master) to the load balancer for external access
+aws elb register-instances-with-load-balancer \
+  --load-balancer-name {{LoadBalancerName}} \
+  --instances ${INSTANCE_ID} \
+  --region {{Region}}
+
 # kubeadm wants lowercase for DNS (as it probably should)
 LB_DNS=$(echo "{{LoadBalancerDns}}" | tr 'A-Z' 'a-z')
 
+# get load balancer ip address to be advertised
+LB_IPV4=$(dig +short {{LoadBalancerDns}})
+
 HOSTNAME="$(hostname -f 2>/dev/null || curl http://169.254.169.254/latest/meta-data/local-hostname)"
 
 # reset kubeadm (workaround for kubelet package presence)
-kubeadm reset
+kubeadm reset --force
```

Comment:
why the --force?

Reply:
Seems without it, it prompts the user for confirmation.